files/default/pyenv.sh

```diff
@@ -0,0 +1,12 @@
+# Prefer a user pyenv over a system wide install
+if [ -s "${HOME}/.pyenv/bin" ]; then
+    pyenv_root="${HOME}/.pyenv"
+elif [ -s "/usr/local/pyenv" ]; then
```

Comment:
This is an attribute defined in defaults. We could make it a template and pass it in, but I also doubt if we'll ever change the system default root.

Reply:
sure thing